autorag/indexer/process/azure/paragraph.py

```diff
@@ -0,0 +1,111 @@
+from llama_index import Document
+from llama_index.schema import TextNode
+from llama_index.node_parser import SentenceSplitter
+
+# Define roles to be excluded
+DEFAULT_EXCLUDED_ROLES: list[str] = ["pageHeader", "pageNumber"]
+
+# Define contents to be excluded
+DEFAULT_EXCLUDED_CONTENTS: list[str] = ["Contains Nonbinding Recommendations"]
+
+DEFAULT_FILE_TYPE: str = "Guidance"
```

Comment:
is it application specific? If it is, maybe move it to config/conf?

Reply:
Considering that we will process product-type documents in the future, I will move DEFAULT_FILE_TYPE to config/conf.